request_a_govuk_domain/settings.py

```diff
@@ -212,6 +212,9 @@
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
 # https://djangosnippets.org/snippets/1303/
-CONTENT_TYPES = ["image"]
-# 2.5 MB
-MAX_UPLOAD_SIZE = "2621440"
+CONTENT_TYPES = ["png", "jpeg", "jpg", "pdf"]
+# 10 MB
+MAX_UPLOAD_SIZE = "104857600"
```

Comment:
Could we reflect this number in the user-facing content? Maybe add another card in the backlog